ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -56,12 +56,15 @@ class BadgeLabel: UILabel, TokenizedControlInternal {
     }
 
     private func updateColors() {
+        let colors = tokenSet.fluentTheme.aliasTokens.colors
```

Comment:
Note: I'm scrubbing through the FUA controls again now to make sure I didn't miss any other spots.